packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1607,11 +1617,19 @@ mixin WidgetInspectorService {
   ///
   ///  * [getChildrenDetailsSubtree], a method to get children of a node
   ///    in the details subtree.
-  String getDetailsSubtree(String id, String groupName) {
-    return _safeJsonEncode(_getDetailsSubtree( id, groupName));
-  }
-
-  Map<String, Object> _getDetailsSubtree(String id, String groupName) {
+  String getDetailsSubtree(
```

Comment:
document what the subtree depth is and that the default is set to 2 for backwards compatibility.

Reply:
done.